src/tools/types.js

```diff
@@ -1,7 +1,8 @@
 export default {
-  external: 'external',
-  dep: 'dependency',
-  ci: 'CI',
-  cli: 'CLI',
-  api: 'api'
+  external: { key: 'external', path: 'external', title: 'Website/GUI' },
+  dep: { key: 'dep', path: 'dependency', title: 'Package' },
+  ci: { key: 'ci', path: 'CI', title: 'CI' },
+  cli: { key: 'cli', path: 'CLI', title: 'CLI' },
+  api: { key: 'api', path: 'api', title: 'API (Web/Server)' },
+  framework: { key: 'framework', path: 'framework', title: 'Framework' }
```

Comment:
Thanks for making things better here :), no reason to hard-code their other details so that was my bad!